setup.py

```diff
@@ -28,7 +28,7 @@
 with open(os.path.join(here, 'requirements-dev.txt')) as requirements_dev_fp:
   dev_install_requires = requirements_dev_fp.read().split('\n')
 
-xgboost_install_requires = ['xgboost>=1.3.1', 'scikit-learn']
+xgboost_install_requires = ['xgboost>=1.3.1', 'scikit-learn', 'numpy']
```

Comment:
fwiw we should at least have minimum versions for this. you don't need to go and find the absolute bare minimum; just use whatever's getting installed and I think that's fine